### DISCUSSION TITLE
Ping is going crazy...

### DISCUSSION THREAD

**Discussion by Aarnid** [jamulussoftware/jamulus#2093] [Technical Support & Help]
Hello, my precedent Jamulus was not working anymore, so i decided to reinstall W10. Now i got an enormous ping with my Xonar Asus Sound Card. I don't understand why coz it was good before, now i got 300ms when i had 50ms max before... 

And when i set my internal motherboard soundcard Realtek the ping is ok, but i don't get any sound of it ! (yes i did plug it right !)

Any ideas would be appreciated ! Thank You !

**Comment by pljones** [comment]
Buffer sizes?

**Reply by Aarnid** [reply]
Nope i tried moving the settings all around, but no improvement :(

**Reply by pljones** [reply]
Have any other ASIO applications seen similar issue?

**Reply by Aarnid** [reply]
When i use Reaper a local sound creative app, i don't have any problem!

**Reply by pljones** [reply]
> When i use Reaper a local sound creative app, i don't have any problem!

And you've checked that the ASIO settings in Jamulus and the ASIO settings in Reaper match?

> i had 50ms max before

That's pretty good as some servers for me are always over 200ms as they're on the other side of the planet...

Can you give examples of _specific_ servers?

Have you checked on Jamulus Explorer to get the IP address and confirmed whether when you use the `ping` command from the console you see similar numbers to what Jamulus shows?

And finally, make sure your mouse pointer isn't over the server list.

**Reply by Aarnid** [reply]
Thanks for the answer pljones.
Yes mi asio set up is the same in Reaper and Jamulus. As i told you i had 50ms before reinstalling windows 10, which is ok to play. But now i got 300ms. The weird thing is that i got 50ms when i switch to my motherboard sound card, but it doesn't work (no sounds at all).
I chose a server from the list like a 43ms server for example Rock Lounge (The ping is 2ms on Jamulus Explorer).
The problem seems to be  : either the driver is not good, or the sound card is working something else behind, or maybe it's the 64 version.
Because when i switch to the motherboard card i got 50ms..
Before i connect to the server it show Rock Lounge 43ms, after i connect it's like this :
![jamulus](https://user-images.githubusercontent.com/93130670/138818240-851a217d-8f62-4b04-a8d4-a01eb2dec01d.jpg)
.

**Comment by pljones** [comment]
I said
> Have you checked on Jamulus Explorer to get the IP address and confirmed whether when you use the ping command from the console you see similar numbers to what Jamulus shows?

(The ping from Jamulus Explorer isn't relevant.)

**Reply by Aarnid** [reply]
OK, i just tried it i got the same result . I remeber that my ASIO didn't show "64" before reinstalling, could it be that ...?
![jamulus2](https://user-images.githubusercontent.com/93130670/138819874-0c95323d-231d-4668-8812-128b34932169.jpg)

**Comment by pljones** [comment]
> OK, i just tried it i got the same result 

Just tried _what_ and got _what_ result?

**Reply by Aarnid** [reply]
Sorry, i mean, i just tried put the IP adress in the server adress to connect. How do you use the ping command from the console ?

**Reply by Aarnid** [reply]
When i use my own server i got nice ping but it's normal i guess
![ownjam](https://user-images.githubusercontent.com/93130670/138822379-d7bc95af-db17-4d0b-a7ba-409a84b77720.png)

**Reply by dcorson-ticino-com** [reply]
That would suggest to me that the problem is at your provider or router, not your computer.

**Reply by Aarnid** [reply]
But if so, why when i switch to my RealTek sound card, my ping is good ?

**Reply by dcorson-ticino-com** [reply]
If that is the case, that would suggest the problem is in your other sound card or it's ASIO driver, not your computer or Jamulus.
Try setting the bit depth to 16bit.  Jamulus doesn't use more.

**Reply by pljones** [reply]
![image](https://user-images.githubusercontent.com/1549463/138915953-6b41e58e-5529-4e5c-8202-0fb454547446.png)
Can you choose in your ASIO Device Settings one of the offered buffer sizes, too.  128, as it says, would be the preferred starting point.

> How do you use the ping command from the console ?

`ping 3.10.235.91`

Now, it may be the server there isn't responding to ICMP ping requests... but if it is, you'll get the network ping time.

**Reply by Aarnid** [reply]
I've set my Asio to 16bits with 10ms.
I cannot access the offered buffer sizes options. They appeared greyed.
How/where do you access the console to type the command line ? I don't find it :/

**Reply by ghost** [reply]
> I've set my Asio to 16bits with 10ms. I cannot access the offered buffer sizes options. They appeared greyed.

Set these parameters if possible in ASIO and/or Jack (default otherwise):
- Soft Mode = checked
- Frames/Period = 128
- Sample rate = 48000 (must be native sample rate, not resampled)
- Periods/Buffer = 2
- Word Length = 16 (Force 16 bit otherwise)
- Audio = Duplex
- Single Application Instance = checked

Do not: Enable Small Network Buffers (In Audio/Network Setup)

**Reply by Aarnid** [reply]
Maybe what you say is under those parameters ? Under Reaper I have set some parameters according to what you ask but i'm not sure since the result in Jamulus doesn'change... Can you tell me what to modify on those panels ?
![jamulusASIO](https://user-images.githubusercontent.com/93130670/138951956-7156cc95-71c4-4f27-bd42-376beab5d532.jpg)
 ?

**Reply by pljones** [reply]
> I've set my Asio to 16bits with 10ms.

Where did you find this?  Do you mean here:
![image](https://user-images.githubusercontent.com/1549463/138957558-d8ef498e-e986-4854-81a2-a9ddd7e42171.png)
What ASIO driver are you using?  It doesn't look like it's very good.  Have you tried replacing it with ASIO4ALL (I rarely suggest that but in this case it could help)?

> I cannot access the offered buffer sizes options. They appeared greyed.

That's _not_ in "ASIO Device Settings", which is where I said to look.  Look in there.

> How/where do you access the console to type the command line ? I don't find it :/

Windows key, then type "cmd".  It should offer "Command prompt".  Choose "Open".  Type "ping 3.10.235.91" and press enter.

**Reply by Aarnid** [reply]
Yes pljones that's the ASIO dedicated box of my soundcard.
Ok i've tried to use only ASIO4ALL and i got better ping/delay but still a bit high, but at least there is improvment here. And i can choose different settings in Buffer Delay (I cannot when i switch to the dedicated ASIO of my card).
By the way when i use "ping 3.10.235.91" in MS-DOS cmd 100% packets are lost dunno why.
It's like C:/Users/Arnid>ping 3.10.235.91 Sent packets 4, lost packets 4.

What do you think i can change in the settings here ?
![jamasus](https://user-images.githubusercontent.com/93130670/139280249-5a58041d-19bb-44cf-ae7a-d6ea0833821b.png)
 :

**Reply by pljones** [reply]
> It's like C:/Users/Arnid>ping 3.10.235.91 Sent packets 4, lost packets 4.

That's not too surprising -- it just means the ICMP packets sent from your PC were not returned from the target server.  Try this one:
```
ping 212.71.237.64
```
That one works for me (I should have checked 3.10.235.91 before suggesting it).  I'm in West London - about 5ms to 7ms from Jamulus Explorer itself.  I get an average ping of about 7ms to `212.71.237.64` (`24 Fighting Camels (` in London).  You're only showing 5ms difference between that and `178.254.43.24` (`DryAged` in Frankfurt), whereas for me it's an extra 12ms (average 19.5ms - which is the 7ms into Central London where Jamulus Explorer lives and then the 12ms it's showing, which makes sense for routing).

Where abouts are you based?  What does the connect dialog show for Any Genre 1?

**Reply by Aarnid** [reply]
I'm in Paris. My problem now is that my ping is ok (like 50-60) when i choose 64 in Buffer Delay but i can't hear the others (i can sometimes hear jitter noise).
And if i choose 128 i can hear them but my ping goes 130ms...
